tuna/miopen/subcmd/export_db.py

```diff
@@ -137,14 +146,33 @@ def get_pdb_query(dbt: MIOpenDBTables, args: argparse.Namespace,
   if args.golden_v is not None:
     src_table = dbt.golden_table
 
-  query = get_base_query(dbt, args, logger)
+  query = get_fdb_query(dbt, args, logger)
   query = query.filter(src_table.params != '')\
       .filter(src_table.solver == dbt.solver_table.id)\
       .filter(dbt.solver_table.tunable == 1)
 
   return query
 
 
+def add_entry_to_solvers(fdb_entry, solvers, logger):
+  """check if fdb_key + solver exists in solvers, add if not present
+  return False if similar entry already exists
+  return True if the fdb_entry is added successfully
+  """
+  fdb_key = fdb_entry.fdb_key
+  if fdb_key not in solvers:
+    solvers[fdb_key] = {}
+  elif fdb_entry.solver in solvers[fdb_key].keys():
+    _ = logger
+    #logger.warning("Skipped duplicate solver: %s : %s with ts %s vs prev ts %s",
+    #                fdb_key, fdb_entry.solver, fdb_entry.update_ts,
+    #                solvers[fdb_key][fdb_entry.solver])
+    return False
```

Comment:
this is a rather odd operation. Is there a better way to do this? maybe just return false? whats the point of line 166?

Reply:
166 silences the unused parameter warning. This statement as a whole is described by the logger message in comment. Presently this would discard duplicate tuning entries if say we are combining results between skews. The entry selected would then depend on the ordering of results, which is set in the fdb query.